engine/src/io/data_parser/GDFParser.h

```diff
@@ -32,6 +33,7 @@ class gdf_parser : public data_parser {
 private:
 	std::vector<cudf::column *> columns;
```

Comment:
these lines shoudl be removed. we no longer keep a cudf::column * here

Reply:
Yep, I was waiting for the William PR for delete those attributes.